### PR TITLE
Adding alert for perf check if db execution takes more that 10 secs

### PR DESCRIPTION
**Pull request by sheetansh2**
Alert added at only single point, we don't need alerts to be sent from multiple position for db execution time so avoiding decorator for now.

### CODE REVIEW

File: sqlserver/datadog_checks/sqlserver/sqlserver.py [hunk 1]
```diff
@@ -585,11 +591,25 @@ def get_sql_type(self, instance, counter_name):
 
     def check(self, instance):
         if self.do_check[self._conn_key(instance, self.DEFAULT_DB_KEY)]:
+            initalTime = time.time()
```
Comment (bhowmickatish) [inline sqlserver/datadog_checks/sqlserver/sqlserver.py]: spellcheck , also probably better to execute the timer after line 596.

File: sqlserver/datadog_checks/sqlserver/sqlserver.py [hunk 2]
```diff
@@ -585,11 +591,25 @@ def get_sql_type(self, instance, counter_name):
...
+
             proc = instance.get('stored_procedure')
             if proc is None:
                 self.do_perf_counter_check(instance)
             else:
                 self.do_stored_procedure_check(instance, proc)
+
+            endTime = time.time()
+            self.log.info("Check for raising alert if it exceeds execution time")
```
Comment (bhowmickatish) [inline sqlserver/datadog_checks/sqlserver/sqlserver.py]: this trace doesnt really add any extra info , make it debug if u want to keep it

File: sqlserver/datadog_checks/sqlserver/sqlserver.py (hunk 1 above)
Comment (sheetansh2) [reply sqlserver/datadog_checks/sqlserver/sqlserver.py]: done

File: sqlserver/datadog_checks/sqlserver/sqlserver.py (hunk 2 above)
Comment (sheetansh2) [reply sqlserver/datadog_checks/sqlserver/sqlserver.py]: made it debug.

File: sqlserver/datadog_checks/sqlserver/sqlserver.py [hunk 3]
```diff
@@ -586,10 +592,23 @@ def get_sql_type(self, instance, counter_name):
...
+            endTime = time.time()
+            self.log.debug("Check for raising alert if it exceeds execution time")
+            if int(endTime - initialTime) >= QUERY_EXECUTION_TIME_LIMIT:
+                self.log.info("Raising alert due to DB execution time exceeded %s secs" % QUERY_EXECUTION_TIME_LIMIT)
+                alert_title = NTNX__db_execution_time_exceeded
+                alert_message = "DB call is taking more than {} secs to execute. Raising Alert".format(QUERY_EXECUTION_TIME_LIMIT)
+                try:
+                    self.raise_alert(instance, alert_title, alert_message)
+                except Exception as e:
+                    self.log.debug("Sending Alert for DB execution time exceed FAILED with exception: %s ", e)
```
Comment (bhowmickatish) [inline sqlserver/datadog_checks/sqlserver/sqlserver.py]: this shud probably b at least a warning since u cudnt raise the alert

**Review by bhowmickatish — APPROVED** [review]

File: sqlserver/datadog_checks/sqlserver/sqlserver.py (hunk 3 above)
Comment (sheetansh2) [reply sqlserver/datadog_checks/sqlserver/sqlserver.py]: sure, done

**Pull request merged**